css/style.css

```diff
@@ -26,4 +27,43 @@ tr:nth-child(even), .highlight{
 
 #store-locations {
     text-indent: -10000px;
+}
+
+h2 {
+    margin-top: 2.3rem;
+    margin-left: 0.625rem;
+}
+
+form {
+    margin: 0.5rem; 
+}
+
+form {
+    width: 400px;
+    padding: 10px;
+    border: 1px solid #773344;
+}
+
+label, input[type="submit"] {
+    margin: 0.5rem 0;
+    display: block;
+}
+
+input[type="submit"] {
+    padding: 0.3rem;
+}
+
+.name-warning-input {
+    outline: none;
+    border-color: #F55;
+    box-shadow: 0 0 3.5px #f55;
+}
+
+#name-warning-message {
+    color: #F55;
+    font-weight: bold;
+}
+
+#hide-name-warning-message {
```

Comment:
A `display: none` class would be better than an `Id`. Something like `.hide`. Toggling Ids is sort of awkward because an element's Id is supposed to be its unique identifier - changing its Id is tantamount to reassigning its unique identifier, which we don't want to do. Classes are better used for CSS styling as a general rule, where Ids are typically useful for getting elements in JavaScript or for web accessibility purposes.